controllers/feeds/getFeedChat.js

```diff
@@ -51,7 +51,8 @@ const getDetail = (req, b, id) => {
   let actor = typeof b.object === 'object' ? b.object.actor : b.actor;
   const isAnonym = typeof b.object === 'object' ? b.object.anonimity : b.anonimity;
   const isOwnSignedPost = actor?.id === id;
-  const isOwnPost = actor?.id === req.userId || actor?.id === id;
+  const isOwnAnonymousPost = actor?.id === anonymousId;
+  const isOwnPost = isOwnAnonymousPost || isOwnSignedPost;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining if a post is owned by the current user has been updated. Previously, it was checking if the actor's id matches either `req.userId` or `id`. Now, it checks if the actor's id matches either `anonymousId` or `id`. This change seems to be in line with the PR's purpose of separating anonymous and signed comment reactions.
```diff
- const isOwnPost = actor?.id === req.userId || actor?.id === id;
+ const isOwnAnonymousPost = actor?.id === anonymousId;
+ const isOwnPost = isOwnAnonymousPost || isOwnSignedPost;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->